docfx.json

```diff
@@ -303,7 +303,7 @@
         "docs/core/**/**.md": "dotnetcontent",
         "docs/core/compatibility/**/**.md": "gewarren",
         "docs/core/deploying/**/**.md": "adegeo",
-        "docs/core/diagnostics/**/**.md": "stmaclea",
+        "docs/core/diagnostics/**/**.md": "sywhang",
```

Comment:
```suggestion
        "docs/core/diagnostics/**/**.md": "suwhang",
```

Reply:
Same for the other ones for ms.author (this one is a Microsoft alias).

Reply:
So the interestinb thing is that sometimes these use github user name,  Others times Microsoft user name.  This should be your MS user name.